Mathlib/RingTheory/Polynomial/Basic.lean

```diff
@@ -180,6 +180,71 @@ theorem eval_eq_sum_degreeLTEquiv {n : ℕ} {p : R[X]} (hp : p ∈ degreeLT R n)
   exact (sum_fin _ (by simp_rw [zero_mul, forall_const]) (mem_degreeLT.mp hp)).symm
 #align polynomial.eval_eq_sum_degree_lt_equiv Polynomial.eval_eq_sum_degreeLTEquiv
 
+theorem degreeLT_eq_degreeLE {n : ℕ} : degreeLT R (n + 1) = degreeLE R n := by
```

Comment:
The current name is probably fine, but `degreeLT_succ` or `degreeLT_succ_eq_degreeLE` might be better.